app/views/Tagging/index.tsx

```diff
@@ -38,6 +46,12 @@ function Tagging(props: Props) {
                 </div>
             </nav>
             <Switch>
+                <Route
+                    exact
+                    path={routes.tagging.path}
+                >
+                    <Redirect to={defaultRoute} />
+                </Route>
```

Comment:
We need to verify if the path should be relative or absolute.

Reply:
Path should be absolute